packages/camera/camera_android_camerax/android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.3.0'
+        classpath 'com.android.tools.build:gradle:7.2.0'
```

Comment:
Can you update the change log in this plugin too? Assuming they are necessary, please also note the version bumps there too!

Reply:
CHANGELOG updated.

Reply:
Does this require a downgrade? Sorry, I must've thought you were upgrading the version here.